dashboard/src/components/Table/PaginationInfo.tsx

```diff
@@ -0,0 +1,94 @@
+import { Table } from '@tanstack/react-table';
+
+import { FormattedMessage } from 'react-intl';
+
+import { MdArrowBackIos, MdArrowForwardIos } from 'react-icons/md';
+
+import { useMemo } from 'react';
+
+import {
+  AccordionItemBuilds,
+  TestByCommitHash,
+} from '@/types/tree/TreeDetails';
+
+import { ItemsPerPageValues } from '@/utils/constants/general';
+
+import { TPathTests } from '@/types/general';
+
+import { Button } from '../ui/button';
+
+import { ItemsPerPageSelector } from './TableInfo';
+
+interface IPaginationInfo {
+  table: Table<TestByCommitHash> | Table<TPathTests>;
+  data: TestByCommitHash[] | TPathTests[] | AccordionItemBuilds[];
+  label: 'builds' | 'boots' | 'tests' | 'trees';
+}
+
+export function PaginationInfo({
+  table,
+  data,
+  label,
+}: IPaginationInfo): JSX.Element {
+  const buttonsClassName = 'text-blue font-bold';
+
+  const pagination = table.getState().pagination;
+  const pageIndex = pagination.pageIndex;
+  const pageSize = pagination.pageSize;
+  const startIndex = pageIndex * pageSize + 1;
+  const endIndex = Math.min((pageIndex + 1) * pageSize, data.length);
+
+  const FormattedLabel = useMemo((): JSX.Element => {
+    switch (label) {
```

Comment:
we should receive the actual string as `label` here, like:

```
<PaginationInfo table={table} data={data} label={intl.formatMessage({ id: 'global.successCount' })} />
```

Reply:
you can use `Extract<MessageKeys>`

Reply:
Done, used MessageKeys as the parameter type and removed the DefaultMessage that was being passed to formattedMessage